editor/src/core/commenting/comment-maintainer.tsx

```diff
@@ -34,6 +35,16 @@ function useMaintainComments() {
   const scenes = useScenes()
   const editThreadMetadata = useEditThreadMetadata()
 
+  const isInteraction = useEditorState(
+    Substores.canvas,
+    (store) => store.editor.canvas.interactionSession != null,
+    'useMaintainComments isInteraction',
+  )
+
+  if (isInteraction) {
+    return
+  }
+
   threads.forEach(async (t): Promise<void> => {
```

Comment:
mhmmmmmm, are we directly calling editThreadMetadata from render? shouldn't this part be wrapped in a useEffect?

Reply:
This triggers an async liveblocks update but does not wait for it, so it did not seem dangerous to me.